src/frontend/web_application/src/styles/object/o-clickable.scss

```diff
@@ -77,9 +77,16 @@
 }
 
 @mixin o-clickable--inline {
-  margin-left: 1px;
-  background: $co-color__fg__back--lower;
-  font-size: $co-font__size--xsmall;
+  display: inline;
+  padding: 0;
```

Comment:
why those reset ?

may be `reset.scss` instead ?

Reply:
mmm I don't think reset.scss is the right place.
The problem here is that `Button` has its own styles (brought by `o-clickable--button`, including padding) and so its modifiers inherit these styles. Modifier `--inline` should not contain any padding nor line-height, etc... so this is the simplest way to override styles only for this modifier.

Reply:
ok so, can I propose you to add the modifier `--inline-block` with `@include o-clickable--button;` to the Button (which will be the default display mode) (Careful to display=expanded which could need it too) ?